ha_lineairdb.cc

```diff
@@ -246,7 +246,7 @@ int ha_lineairdb::delete_row(const uchar*) {
   LineairDB::TxStatus status;
```

Comment:
If you have time, remove declarations of `status` variables at each location. These are unused variables.